packages/ui-components/example/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "example",
```

Comment:
Was the `ui-components/example/` folder meant to be included in Git versioning as well?  Or can it be removed?